### DISCUSSION TITLE
How to query on table relation?

### DISCUSSION THREAD

**Discussion by o8e** [supabase/supabase#1554] [Questions] [answered]
I can't see anything in the docs to suggest you can do this, and attempting the obvious ways doesn't seem to give me much joy.

```
return this.$supabase
  .from('posts')
  .select(`*,meta:posts_tracks(tracks(*))`)
  .gte('???????.tracks.duration', 600000)
```

I need something like this. I have got it returning the right results in the `tracks` array, but it still returns all the posts, just with if they don't match:

```
"meta": [
  {
    "tracks": null
  }
]
```

Any thoughts? Cheers.

**Comment by easy-street** [comment, marked as answer]
i believe this is a known issue. from: https://github.com/supabase/supabase/issues/875#issuecomment-799592556

> A workaround would be to filter the nulls on client side.

**Reply by o8e** [reply]
Thank you. That's a shame, filtering on the client means that you can't really paginate properly. I'll keep an eye on that issue then.

**Reply by awalias** [reply]
yes this is dependent on https://github.com/PostgREST/postgrest/issues/1075
